wgpu-core/src/resource.rs

```diff
@@ -787,8 +794,8 @@ pub struct DestroyedBuffer {
     device: Arc<Device>,
     label: String,
     bind_groups: WeakVec<BindGroup>,
-    #[cfg(feature = "indirect-validation")]
     raw_indirect_validation_bind_group: Option<Box<dyn hal::DynBindGroup>>,
+    raw_indirect_draw_validation_bind_group: Option<Box<dyn hal::DynBindGroup>>,
```

Comment:
Similar nit to module naming, I feel like we should have naming parity here - draw_validation and compute_validation

Reply:
I ended up combining those 2 in the new module.